### PR TITLE
add py-cwl-registry and py-cwl-luigi

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyCWLRegistry(PythonPackage):
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]:
```suggestion
class PyCwlRegistry(PythonPackage):
```

File: bluebrain/repo-bluebrain/packages/py-cwl-luigi/package.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyCWLLuigi(PythonPackage):
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-cwl-luigi/package.py]:
```suggestion
class PyCwlLuigi(PythonPackage):
```

File: bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py (hunk 1 above)
Comment (mgeplf) [reply bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: even when CWL is an acronym?

Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: yes, or your package will be `py-c-w-l-registry` :D

Comment (mgeplf) [reply bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: lol, yes, then I will change it, thanks for the heads up

File: bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    depends_on('python@3.7:', type=('build', 'run'))
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools-scm', type='build')
+
+    depends_on("py-click@8.0.0:", type='run')
+    depends_on('py-voxcell', type='run')
+    depends_on('py-numpy', type='run')
+    depends_on('py-pandas', type='run')
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: All Python dependencies should be `("build", "run")` by now.

File: bluebrain/repo-bluebrain/packages/py-cwl-luigi/package.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    homepage = "https://bbpgitlab.epfl.ch/nse/cwl-luigi"
+    git = "ssh://git@bbpgitlab.epfl.ch/nse/cwl-luigi.git"
+
+    version('develop', branch='main')
+    version('0.3.0', tag='cwl-luigi-v0.3.0')
+
+    depends_on('python@3.9:', type=('build', 'run'))
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-setuptools-scm', type=('build', 'run'))
```
Comment (eleftherioszisis) [inline bluebrain/repo-bluebrain/packages/py-cwl-luigi/package.py]:
```suggestion
```
Not used.

File: bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    homepage = "https://bbpgitlab.epfl.ch/nse/cwl-registry"
+    git = "ssh://git@bbpgitlab.epfl.ch/nse/cwl-registry.git"
+
+    version('develop', branch='main')
+    version('0.3.0', tag='cwl-registry-v0.3.0')
+
+    depends_on('python@3.9:', type=('build', 'run'))
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-setuptools-scm', type=('build', 'run'))
```
Comment (eleftherioszisis) [inline bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]:
```suggestion
```
Not used.

**Review by eleftherioszisis — APPROVED** [review]

**Review by matz-e — APPROVED** [review]

**Pull request merged**